src/dev-core/src/app/button/button.demo.scss

```diff
@@ -0,0 +1,4 @@
+.inverse-btn-container {
+  padding: 0.5rem 0.5rem;
```

Comment:
We probably want to use the rem sizing variables here....

This padding translates to 10px in 3.x...

Or maybe it's not a big deal?

Reply:
When should we use rem sizing variables vs just using rem directly?

Reply:
This is for the demo application and not the component library. It basically houses the inverse buttons so we can see the white buttons against a dark background.

I don't think we need to use the rem sizing variables for this page. Otherwise I need to import Clarity variables into the demo page, which I think is overkill. 

Resolving for now. Let me know if you disagree!